src/site-config.ts

```diff
@@ -1,13 +1,12 @@
 import { SiteMeta } from './interfaces';
-import { RouteConfig } from '@dojo/framework/routing/interfaces';
 
 export const meta: SiteMeta = {
 	title: 'learn dojo',
 	author: 'odoenet',
 	description: 'byte by byte',
 	footerLinks: [
 		{
-			href: '/atom.xml',
+			href: 'https://learn-dojo/atom.xml',
```

Comment:
Made this fully qualified as it was being picked up by the auto discovery in BTR, I didn't think it would be an issue (but could be wrong)